.github/workflows/aws-docker-deploy.yml

```diff
@@ -36,8 +43,18 @@ jobs:
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
 
+      - name: Fill .env file for frontend-client
+        if: ${{ inputs.client-environment-file-path }}
+        shell: bash
+        run: |
+          echo ${{ inputs.dotenv-value }}
+          touch ${{ inputs.client-environment-file-path }}
+          echo -e "${{ inputs.dotenv-value }}" > ${{ inputs.client-environment-file-path }}
+          echo ".env created!"
+          cat ${{ inputs.client-environment-file-path }}
+
       - name: Build, tag and Push image
-        uses: ingeno/aws-workflows/.github/actions/docker-deploy@v1
+        uses: ingeno/aws-workflows/.github/actions/docker-deploy@v3
```

Comment:
There's probably something I don't understand in how actions are deployed, but how could you refer to a new version of the docker-deploy action in this repo, if this PR modifies only the AWS Docker deploy action ?

Is is because once this PR gets merged, it releases new versions of all actions in this repo, regardless of changes ?

Reply:
This repo's workflows do not use the v3 action, so that's why it does not implode. The reference is also not validated by github, up until actual use, which only happens during the greenfield's workflow execution. 

These actions are not really "deployed", we just create a tag that greenfield (or any other workflow) can refer to.

A bit hard to explain clearly in text, but I hope that helps. If not, we can meet.